css/styles.css

```diff
@@ -0,0 +1,17 @@
+.body {
+  display: sticky;
+  top: 0;
+  flex-direction: column;
+  background-color: rgb(78, 82, 83);
+}
+.header {
```

Comment:
for header, you don't need to make a new class. you can use the existing one <header>.